cdap-examples/SportResults/src/main/java/co/cask/cdap/examples/sportresults/SportResults.java

```diff
@@ -51,11 +51,11 @@ public void configure() {
     createDataset("totals", PartitionedFileSet.class, PartitionedFileSetProperties.builder()
       // properties for partitioning
       .setPartitioning(Partitioning.builder().addStringField("league").build())
-        // properties for file set
+      // properties for file set
```

Comment:
John, developers typically do not start comments in upper case. If you want to change that, we should do a pass over all the examples. I can change it here, but how pedantic do we need to be here?